Woof/Woof/View/OwnerFlow/BookingSlotView/BookingSlotView.swift

```diff
@@ -2,15 +2,68 @@ import SwiftUI
 
 /// A view presents the information related to booking process.
 struct BookingSlotView: View {
+    /// The view model responsible for providing data to the view.
+    @ObservedObject var viewModel: BookingSlotViewModel
```

Comment:
Why have you decided to remain `@ObservedObject` here?

Reply:
Because it's injected as a parameter,  and we need a walking to initialize it. If you can provide the solution on how to do this I will be happy to know 🥲

Reply:
As discussed, refactor the view with https://github.com/IronFoundation/iOS-Woof/pull/112/commits/c420cf3d40c9ad92196da28e5f3a6b86981629b5